package.json

```diff
@@ -46,10 +46,7 @@
   "contributors": [
     "Suzanne Rozier <suz@truss.works>",
     "John Gedeon <john@truss.works>",
-    "Erin Stanfill <erin@truss.works>",
-    "Sara Bocciardi <sara@truss.works>",
-    "Jeri Sommers <sojeri@truss.works>",
-    "Emily Mahanna <emahanna@truss.works>",
```

Comment:
I understand that the four people above are no longer working at Truss, but taking a look at the history of the repository, these folks did contribute to the project. The property here is `contributors` and not `active_contributors` so I'm not sure if removing them makes sense. But I'm okay to treat this as `active_contributors` if that's what the goal of this change is.
cc: @Igarfinkle

Reply:
Oh, of course! My mistake, we should absolutely include them.

Reply:
fixed